src/ForwardPagination.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Spatie\JsonApiPaginate;
+
+use Illuminate\Database\Eloquent\Builder;
+use Illuminate\Database\Eloquent\Relations\BelongsToMany;
+use Illuminate\Database\Eloquent\Relations\HasManyThrough;
+use Illuminate\Support\Arr;
+
+class ForwardPagination
+{
+    public static function execute($builder, int $maxResults = null, int $defaultSize = null) {
```

Comment:
I think it would be nice to have this as a non-static method. Maybe it'd be nice to have a constructor to inject the `Builder` instance, and have a public `execute` method for the actual execution of the pagination.